spring-apps/autosleep-core/src/main/java/org/cloudfoundry/autosleep/ui/web/model/OrgEnrollmentConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autosleep.ui.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class OrgEnrollmentConfigRequest {
+
+    @JsonProperty("idle-duration")
+    private String idleDuration;
+
+    @JsonProperty("exclude-spaces-from-auto-enrollment")
+    private String excludeSpacesFromAutoEnrollment;
+
+    @JsonProperty("state")
+    private String state;
```

Comment:
Could we find a way to map this as enum and keep this consistent with EnrollmentState https://github.com/cloudfoundry-community/autosleep/blob/f890f347e0a8de72db9c61e956859434011ba5b4/common/src/main/java/org/cloudfoundry/autosleep/config/EnrollmentConfig.java#L5-L14 ?